### DISCUSSION TITLE
New Refit Version Incoming

### DISCUSSION THREAD

**Discussion by anaisbetts** [reactiveui/refit#1494] [Announcements]
In the past few releases which have been unlisted, we accidentally merged some PRs that have some issues - we unlisted the affected Refit versions but it'll take a minute to get an updated version online.

There _will_ be breaking changes from the unlisted versions, though we're going to try to do our best to still keep the Spirit:tm: of all of the features merged as much as possible if we can't keep it as-is. Sorry about the churn / miscommunication, and we'll try to keep this thread up-to-date as much as possible.

We'll also try to get the latest .NET support in for the latest release as well, so that's a carrot to come along with the hopefully slight annoyance of possibly having to change some code / plans :) 

Follow along here:

* #1496 
* #1495

**Comment by squadwuschel** [comment]
Hi,
could you tell us when the next release could be published on NuGet?

**Reply by anaisbetts** [reply]
When I finish #1496, which has been delayed due to family issues

**Comment by anaisbetts** [comment]
For those who want to give this a try, v7.0.0-beta.1 is headed to NuGet, if that works for everyone I think we can ship v7.0 soon (don't get scared off by the major version bump, there will likely be very little / no changes required to your app)

**Reply by marcominerva** [reply]
I have used Refit v7.0.0-beta1 during the last week and everything works as expected.

**Reply by marcominerva** [reply]
Hi @anaisbetts, have you any updates on the release of v7.0? Thank you!

**Comment by glennawatson** [comment]
This announcement is long obsolete. Refit v7.0 shipped, and current releases are well beyond it (v8+). Closing this tracking thread as outdated. For the latest, see the NuGet package and the releases page.
